app/src/components/ArtistEvents.jsx

```diff
@@ -1,32 +1,40 @@
 import { useState } from "react";
```

Comment:
nitpick but you can use import{useState,useEffect} from 'react'; or something similar, not exactly sure what the separating char is